TaskManagementSystem/main.go

```diff
@@ -38,8 +69,7 @@ func main() {
 	// Third print
 	fmt.Println("Project: Task Management System")
 	fmt.Println()
-	session2()
+	TaskUpdate(95)
+	fmt.Println()
+	fmt.Println()
```

Comment:
No need to extra lines, as of TaskUpdate() function last output makes \n inside the message